remotepin.go

```diff
@@ -0,0 +1,18 @@
+package config
+
+const (
+	PinningTag             = "Pinning"
+	RemoteServicesTag      = "RemoteServices"
+	RemoteServicesSelector = PinningTag + "." + RemoteServicesTag
+)
+
+type Pinning struct {
+	RemoteServices map[string]RemotePinningService
+}
+
+type RemotePinningService struct {
+	ApiEndpoint string
+	ApiKey      string
```

Comment:
WDYT about moving these into an Api sub struct, it'll allow us to mess around with the fields here later with less noise?